### PR TITLE
Makes the Sani'Orios away site more interesting

### PR DESCRIPTION
**Pull request by Alberyk**
The Sani'Orios away site now has a dpra asteroid outpost with a small crew and some defenses.

### CODE REVIEW

**Comment by DreamySkrell** [comment]
image
missing wire

**Comment by DreamySkrell** [comment]
image
wrong turf here (space)

**Review by DreamySkrell — CHANGES_REQUESTED** [review]
.

File: maps/away/away_site/tajara/saniorios_outpost/saniorios_outpost_ghostroles.dm [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/datum/ghostspawner/human/saniorios_outpost_crew
+	short_name = "saniorios_outpost_crew"
+	name = "Sani'Orios Asteroid Outpost Crewmember"
+	desc = "Crew a Democratic People's Republic of Adhomai's asteroid outpost in Sani'Orios."
```
Comment (DreamySkrell) [inline maps/away/away_site/tajara/saniorios_outpost/saniorios_outpost_ghostroles.dm:4]: desc should explain more about this role - like what do they do?
there should also be welcome_text or whatever, that shows on spawn, and explains things more in detail

**Review by NonQueueingMatt — APPROVED** [review]

**Pull request merged**